back/src/main/java/sellerlee/back/article/domain/Article.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * @author jnsorn
+ */
+
+package sellerlee.back.article.domain;
+
+import sellerlee.back.member.domain.Member;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+public class Article {
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @NotBlank(message = "제목을 입력해 주세요.")
+    private String title;
+
+    @Size(min=0, message = "0원 이상의 금액을 입력해 주세요.")
+    private Long price;
+
+    @Enumerated(EnumType.STRING)
+    private Category category;
+
+    @NotNull(message = "내용은 null이 될 수 없습니다.")
+    @Lob
+    private String contents;
+
+    @ManyToOne
+    private Member member;
+
+    @Embedded
+    private Tags tags;
+
+    protected Article() {
+    }
+
+    public Article(String title, Long price, Category category, String contents, Member member, Tags tags) {
+        this.title = title;
+        this.price = price;
+        this.category = category;
+        this.contents = contents;
+        this.member = member;
+        this.tags = tags;
+    }
+
+    public Article(Long id, String title, Long price, Category category, String contents, Member member, Tags tags) {
+        this(title, price, category, contents, member, tags);
+        this.id = id;
+    }
```

Comment:
여기 두 생성자의 위치가 바껴야 할 것 같아요.
모든 필드를 인자로 받는 생성자가 위에 있고 해당 생성자 내부가 모든 필드를 직접 초기화하는 방법으로, 그리고 id를 제외한 필드를 받는 생성자를  this 생성자를 사용하여 구현하는 것이 더 옳은 모양일 것 같아요 👍 
코드 참고하시라고 suggestion 남겨두었습니다 :)

```suggestion
    public Article(Long id, String title, Long price, Category category, String contents, Member member, Tags tags) {
        this.id = id;
        this.title = title;
        this.price = price;
        this.category = category;
        this.contents = contents;
        this.member = member;
        this.tags = tags;
    }

    public Article(String title, Long price, Category category, String contents, Member member, Tags tags) {
        this(null, title, price, category, contents, member, tags);
    }
```

Reply:
좋은 의견 고마워요! 회의 때 얘기한 대로 주 생성자를 위쪽에 위치시키고, 매개변수가 적은 생성자는 주 생성자에 null값을 전달하도록 수정하겠습니다.